notebooks/individual-dashboards/fairness-dashboard/fairness-dashboard-loan-allocation.ipynb

```diff
@@ -75,7 +75,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data = fetch_openml(data_id=1590, as_frame=True)\n",
+    "from raiutils.common.retries import retry_function\n",
+    "\n",
+    "class FetchOpenml(object):\n",
```

Comment:
Why do we need the `FetchOpenml` class? Seems like we need to provide the reference to the function in function `retry_function`?